app/controllers/orders_controller.rb

```diff
@@ -4,7 +4,10 @@ class OrdersController < ApplicationController
   before_action :set_orders, only: [:index]
   before_action :set_order, only: [:show]
 
-  def index; end
+  def index
+    @q = current_user.orders.ransack(params[:q])
+    @orders = @q.result(distinct: true).paginate(page: params[:page], per_page: 10)
```

Comment:
The same, `@q` variable name.

Reply:
Why do we have a constant of 10 and we don't take it from params? Same as in products. I prefer do helpers for sorting and pagination. I have sent you an example.